readme.md

```diff
@@ -10,14 +10,30 @@ Written by Taylor Otwell as a part of Laravel's [Illuminate/Support](https://git
 
 Lovingly split by Matt Stauffer for [Tighten Co.](https://tighten.co/), with a kick in the butt to finally do it from [@assertchris](https://github.com/assertchris).
 
-## Installation
+## Install
 
 With [Composer](https://getcomposer.org):
 
 ```bash
 composer require tightenco/collect
 ```
 
+## Upgrade
```

Comment:
Maybe rename "Upgrade" to "Development" ? or something like that?

Reply:
I think Development makes a lot more sense, because someone could be tasks with something other than upgrading Collect. The first sentence now tackles upgrading. Thank you for the re-write session.